app/actions/remote/post.ts

```diff
@@ -226,6 +226,8 @@ export const fetchPostsForChannel = async (serverUrl: string, channelId: string,
         }
 
         if (!fetchOnly) {
+            const isCRTEnabled = await getIsCRTEnabled(operator.database);
```

Comment:
this whole thing is being done when `fetchOnly` is false.. but in the websocket action `handleUserAddedToChannelEvent` we call this with `fetchOnly` true, do we need to handle this there or are we ok with how it is?

Reply:
We should be OK, since `handleUserAddedToChannelEvent` fetches `my_channels` so all data will be up to date.